site/users/index.html.js

```diff
@@ -9,672 +9,11 @@
-  },
-  {
-    name: 'ComparaOnline',
-    img: 'comparaonline.png',
-    link: 'https://www.comparaonline.com'
-  },
-  {
-    name: 'Goalify.plus',
-    img: 'goalify.png',
-    link: 'https://goalify.plus'
-  },
-  {
-    name: 'Teachers Pay Teachers',
-    img: 'teacherspayteachers.png',
-    link: 'https://teacherspayteachers.com'
-  },
-  {
-    name: '1stdibs',
-    img: '1stdibs.png',
-    link: 'https://www.1stdibs.com/'
-  },
-  {
-    name: 'Hackages',
-    img: 'hackages.png',
-    link: 'https://hackages.io/'
-  },
-  {
-    name: 'Fairfax Media',
-    img: 'fairfaxmedia.png',
-    link: 'https://www.fairfaxmedia.com.au'
-  },
-  {
-    name: 'Amplitude',
-    img: 'amplitude.png',
-    link: 'https://amplitude.com/engineering'
-  },
-  {
-    name: 'Gentux',
-    img: 'gentux.png',
-    link: 'https://www.generationtux.com'
-  },
-  {
-    name: 'Bynder',
-    img: 'bynder.png',
-    link: 'https://www.bynder.com'
-  },
-  {
-    name: 'Hasura',
-    img: 'hasura.png',
-    link: 'https://hasura.io'
-  },
-  {
-    name: 'Atlassian',
-    img: 'atlassian.png',
-    link: 'https://www.atlassian.com'
-  },
-  {
-    name: 'Lets.events',
-    img: 'letsevents.png',
-    link: 'https://lets.events'
-  },
-  {
-    name: 'KLM Royal Dutch Airlines',
-    img: 'klm.png',
-    link: 'https://www.klm.com'
-  },
-  {
-    name: 'Universe',
-    img: 'universe.png',
-    link: 'https://www.universe.com'
-  },
-  {
-    name: 'StackShare',
-    img: 'stackshare.png',
-    link: 'https://stackshare.io'
-  },
-  {
-    name: 'Metric.ai',
-    img: 'metric-ai.png',
-    link: 'https://www.metric.ai'
-  },
-  {
-    name: 'Jusbrasil',
-    img: 'jusbrasil.png',
-    link: 'https://www.jusbrasil.com.br'
-  },
-  {
-    name: 'NBC News Digital',
-    img: 'nbc-news-digital.png',
-    link: 'https://www.nbcnews.com'
-  },
-  {
-    name: 'PayPal',
-    img: 'paypal.png',
-    link: 'https://www.paypal.com/'
-  },
-  {
-    name: 'Cheddar',
-    img: 'cheddar.png',
-    link: 'https://cheddar.com'
-  },
-  {
-    name: 'UC Trends',
-    img: 'uctrends.png',
-    link: 'https://trends.ucweb.com'
-  },
-  {
-    name: 'Expert360',
-    img: 'expert360.png',
-    link: 'https://expert360.com'
-  },
-  {
-    name: 'Cloverleaf',
-    img: 'cloverleaf.png',
-    link: 'https://cloverleaf.me/'
-  },
-  // Adding your logo?
-  // Add it to the /users/logos/ directory and then append an entry above this comment.
-  //
-  // Please include logos with transparent backgrounds with no extra margin in the image.
-  // Images should be around 400px in the larger dimension.
-  // If your logo is round, include `isRound: true` in your entry.
-  //
-  // Please leave a trailing comma after your entry to avoid merge conflicts.
-];
+const iframe = '<iframe frameBorder="0" id="landscape" scrolling="no" style="width: 1px; min-width: 100%" src="https://landscape.graphql.org/category=graph-ql-adopter&format=card-mode&grouping=no&format=logo-mode&embed=yes"></iframe>';
```

Comment:
@caniszczyk please, replace `format=logo-mode` with `format=card-mode` this line. That is all you need, thanks!

Reply:
thanks